### ISSUE TITLE
Sketch does not cover windows on retina display

### ISSUE THREAD

**Issue by swirly** [p5py/p5#131]
On a retina display, the skecth is only covering a quarter of the window. This is a reopening of #9 since the supposed fix does not seem to work

In order to reproduce, use a retina display and just launch a script. It should cover the entire windows, it does not

Here is a simple screenshot
<img width="721" alt="Capture d’écran 2019-10-02 à 12 41 21" src="https://user-images.githubusercontent.com/2655257/66038631-fc811200-e512-11e9-9501-b933de832926.png">

**System information:**
 - P5 Version: 0.6.0 (latest version, installed via pip 19)
 - Python version: Python 3.7.3 (anaconda)
 - Operating system: Mac OS X Mojave

**Comment by jeremydouglass** [comment]
In order to run at double density with a retina display, it seems like (untested) you need to set 

```
builtins.pixel_x_density = 2
builtins.pixel_y_density = 2
```

This would be the equivalent of [displayDensity()](https://processing.org/reference/displayDensity_.html)

Did you try setting these manually? Auto-detection would be better, but the first question is whether density is working at all....

**Comment by swirly** [comment]
Thank you for your investigation

just tried this now
```
from p5 import *
from random import random, randrange
import builtins

builtins.pixel_x_density = 2
builtins.pixel_y_density = 2
```

Does not solve anything. Tried to use 1 or 4. The behaviour does not change. Density seems not to work

**Comment by swirly** [comment]
Any progress on this issue ?

**Comment by pikeas** [comment]
I'm on a Retina MBP and can confirm that this appears to be working correctly with p5 0.7.1.

**Comment by ziyaointl** [comment]
Closed since this is working in 0.7.1.
